### ISSUE TITLE
return true for some RGB images

### ISSUE THREAD

**Issue by QuentinGprd** [zengming00/node-jpg-is-cmyk#1]
Hello,

First of all, thanks for the module, it is working in some cases.

However, I have a false positive with a lot of images, it returns true at the following conditional:
```
    // CMYK
    if (data.includes('CMYK')) {
        return true;
    }
```
I have no idea how finding 'CMYK' in the Buffer works so any help would be greatly appreciated. Maybe there is a way to make it more restrictive?

You can find an RGB image that returns true here (zipped to prevent any image conversion on upload):
[omer.jpg.zip](https://github.com/zengming00/node-jpg-is-cmyk/files/3641789/omer.jpg.zip)

Thanks for your help.

**Comment by zengming00** [comment]
Thank you for your report. As you can see, this module has not been maintained for two years. I have not done any related work, so I may not be able to give you much help. I simply deleted the lines of code and then released Version 1.0.2, It should work fine.
Direct search for the 'CMYK' string may be a simple way to determine if a picture is CMYK. Unfortunately, this doesn't work. I haven't found this problem because I will convert CMYK images to RGB using tools like ffmpeg, so there is This issue has not been affected as a result.

**Comment by QuentinGprd** [comment]
Yep, I have also deleted this part and after some tests, cmyk files are returning true later in the code anyway.

I will probably end doing the conversion automatically for the user too, when I have more time.

Thank you!
